### DISCUSSION TITLE
HOW CAN I IGNORE SOME TESTCASES RESULT IN Pytest AND Allure?

### DISCUSSION THREAD

**Discussion by Neko-chanQwQ** [allure-framework/allure2#2636] [Questions & Support]
For Example, some testcase depend some api request, these testcase run after api request
Because some problem , the depend request need be testcase in the same testclass and use pytest.mark.parametrize
so when pytest run these depend testcase, it will record the testcase result and allure also record, it may look not good
any ways can ignore these depend testcase result?

**Comment by delatrie** [comment]
Hi, @Neko-chanQwQ 
Could you please provide a code example?

**Comment by Neko-chanQwQ** [comment]
![image](https://github.com/user-attachments/assets/50c885b0-7a1f-4f89-b112-7cf0109f343c)
@delatrie For example, test_dependance_orderconfirmed is a prerequisite step for test_order_cancel. It can also be regarded as a regular test case. Because I need to input the requested information into the fixture for signature header generation, the preceding steps can only be made into regular use cases.
The effect I want now is that the result of executing the previous test case test_dependance_order_comfirm will not be calculated in the prometheusData.txt of allure. This way, when I view the overall report, the actual number of running use cases will not be inconsistent due to operations like this.
